CHANGELOG.md

```diff
@@ -2,11 +2,51 @@
 
 ## Table of Contents
 
+- **[r1.2](#r12)**
 - [r1.1](#r11)
 - [v0.1.0](#v010)
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# r1.2
+## Release Notes
+
+This release contains the definition and documentation of
+* population-density-data v0.1.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
+
+
+## population-density-data v0.1.1
+
+**population-density-data v0.1.1 is the public release of the version 0.1.1**
+
+v0.1.1 is a patch release of v0.1.0 which includes only changes of the info object in alignment with Commonalities 0.4.0, completing the alignment with Commonalities v0.4.0 and Consent Management v0.2.0 guidelines included in the CAMARA Fall24 meta-release. Test cases are included for API validation. No new features were included.
+
+- API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/r1.2/code/API_definitions/population-density-data.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/r1.2/code/API_definitions/population-density-data.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/r1.2/code/API_definitions/population-density-data.yaml)
+
+### Added
+* Include  `x-camara-commonalities` by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/41
+* Include testing definitions in .feature file for Fall24 meta-release by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/44
+* Included documentation for the initial posible implementation algorithm by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/25
+* Included API test plan for Population Density Data by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/44
+
+### Changed
+* Updated Authorization and authentication section according to I&CM last release (v0.2.0-rc1) by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/41
```

Comment:
The correction you applied in #41 was the corrected one from ICM v0.2.0-rc2, not v0.2.0-rc1.

```suggestion
* Updated Authorization and authentication section according to I&CM latest release by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/41
```